src/main/resources/application-dev.yml

```diff
@@ -23,6 +23,9 @@ familie:
     integrasjoner:
       url: https://familie-integrasjoner.dev-fss-pub.nais.io/api
       scope: api://dev-fss.teamfamilie.familie-integrasjoner/.default
+  pdf:
+    url: https://familie-pdf.intern.dev.nav.no/api
+
```

Comment:
Test gjerne at denne fungerer i preprod før merge til main. Denne verdien må også legges inn for application-prod.yml